### PR TITLE
fix parse bug

### PR DESCRIPTION
**Pull request by decker502**
字段内容中包含 ";"时，会触发解析bug，导致解析的Sql语句不完整。
例如：
INSERT INTO test  (id, title, content, tenum, tset) VALUES (?, ?, ?, ?, ?)", 1, "first", "hello mysql; 2", "e1", "a,b")

### CODE REVIEW

**Comment by siddontang** [comment]
Thanks @decker502 

Can you add some tests for it?

**Comment by decker502** [comment]
@siddontang done

File: dump/parser.go [hunk 1]
```diff
@@ -50,7 +50,9 @@ func Parse(r io.Reader, h ParseHandler, parseBinlogPos bool) error {
 		}
 
 		// Ignore '\n' on Linux or '\r\n' on Windows
-		line = strings.SplitAfter(line, ";")[0]
+        line = strings.TrimFunc(line, func(c rune) bool {
```
Comment (siddontang) [inline dump/parser.go]: can you format your code?

Comment (siddontang) [reply dump/parser.go]: here TrimRightFunc may be better?

Comment (decker502) [reply dump/parser.go]: No problem

**Pull request merged**